expected_output = { 
   "showing_sub_info":1,
   "state":"UNREACHABLE",
   "last_reset_reason":"UNKNOWN REASON",
   "uart_fe_error":0,
   "uart_pe_error":0,
   "uart_dor_error":0,
   "rx_buf_overflow":0,
   "rx_buf_underflow":0,
   "tx_buf_full":0,
   "rx_bad_endbyte":0,
   "ple_invalid_app":0,
   "ple_disabled_app":0,
   "ple_invalid_data":0,
   "ple_invalid_flags":0,
   "ple_app_error":0,
   "ple_lost_ctxt":0,
   "ple_invalid_reg":0,
   "ple_invalid_reg_len":0,
   "ple_invalid_msg_len":0,
   "sle_poe_no_port":0,
   "sle_invalid_reg_len":0,
   "sle_msg_underrun":0
}
